k2/csrc/dtype.h

```diff
@@ -14,26 +14,26 @@
 
 #include <cstdint>
 
-#include "k2/csrc/log.h"
+#include "k2/csrc/log.cuh"
 
 namespace k2 {
 
 enum BaseType {      // BaseType is the *general type*
   kUnknownBase = 0,  // e.g. can use this for structs
   kFloatBase = 1,
-  kIntBase = 2,   // signed int
-  kUintBase = 3,  // unsigned int
+  kIntBase = 2,   // signed int32_t
```

Comment:
I think these two comments are not correct. For `kIntBase`, it also denotes `int8_t`,`int16_t`, and `int64_t`. The same goes for `kUintBase`.

Reply:
Hi @danpovey, could you help here? IMO, it's designed so, as the `g_dtype_traits_array` distinguishes each other.

Reply:
should really be "signed integer type", "unsigned integer type"